patitasuy/src/app/pages/profile/profile.component.html

```diff
@@ -1,9 +1,63 @@
-<ion-header></ion-header>
+<ion-header class="bar">
+  <mat-icon class="container-iconPets">pets</mat-icon>
+  <ion-title class="title">Perfil</ion-title>
+</ion-header>
+
 <ion-content>
-  <p>
-    profile works!
-  </p>
+  <div class="userContent">
+    <ion-icon class="container-iconAccount" name="person"></ion-icon>
+    <div class="userDates">
+      <p class="userName">{{name}}</p>
+      <p class="userEmail">{{email}}</p>
+    </div>
+    <ion-button id="open-modal" class="editButton" expand="block">
+      <ion-icon name="create-outline"></ion-icon>
+    </ion-button>
+    <ion-modal trigger="open-modal">
```

Comment:
Consider moving this modal to a separate component
pros:
- More organized, cleaner code
- Easier to mantain

cons:
- You will have to use Input() and Output(), which I am not sure you have used before

Reply:
Ok its a good change, I implemented it